docs/web/concepts-advanced-config.mdx

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
Shall we rename this file to `concepts-advanced-app-config` for consistency with the `concepts-app-config` equivalent?

Reply:
Totally. Done.